src/navigatorContextMenu.jsx

```diff
@@ -47,12 +47,16 @@ export const ContextMenu = ({ parentId, contextMenuItems, setSelectedContext })
         };
 
         const parent = document.getElementById(parentId);
-        parent.addEventListener("contextmenu", _handleContextMenu);
-        document.addEventListener("click", _handleClick);
+        if (parent && parent.id === "folder-view") {
```

Comment:
Can you please add an comment here for why the 'if' is needed?

Reply:
Without it, we would try to remove the event listener when 'parent' doesn't exist. This breaks multi-selection though, I will look at it again later